src/controllers/api/ruleSectionController.js

```diff
@@ -0,0 +1,35 @@
+const RuleSection = require('../../models/ruleSection');
+const utility = require('./utility');
+
+exports.index = async (req, res, next) => {
+  const search_queries = {};
+  if (req.query.name !== undefined) {
+    search_queries.name = { $regex: new RegExp(utility.escapeRegExp(req.query.name), 'i') };
+  }
+  if (req.query.desc !== undefined) {
```

Comment:
`req.query.description` might make more sense? We don't have a character limit.

Reply:
The field on the data object is `desc`. I think it would make sense for them to match. All the other descriptions are named `desc` and I figured it should be consistent. I can just support both if you want

Reply:
No. You're right. I guess I'm just unhappy about that field name but that's not something you need to fix.